### ISSUE TITLE
Fatal Error: undefined method `to_sym' for 404:Integer

### ISSUE THREAD

**Issue by CAMOBAP** [relaton/relaton-w3c#14]
During packed-mn release we stable getting errors:

```
[relaton-w3c] ("W3C XMLSchema") fetching...
Fatal Error: undefined method `to_sym' for 404:Integer
```

```
[relaton-w3c] ("W3C REC-xml") fetching...
Fatal Error: undefined method `to_sym' for 404:Integer
```

Full log https://github.com/metanorma/packed-mn/actions/runs/1466474915

**Comment by ronaldtse** [comment]
@CAMOBAP permissions also fixed.

**Comment by andrew2net** [comment]
The relaton-w3c is changed to fetch document from the repo https://github.com/relaton/relaton-data-w3c, so references should be exactly as the file names in the repo. For example the `W3C XMLSchema` should be `REC-XMLSCHEMA-2-20041028` and `W3C REC-xml` should be `REC-XML-20081126`.
I'll update the script to warn then reference isn't found.

**Comment by andrew2net** [comment]
fixed in v 1.9.4
